tests/UploadControllerTest.cs

```diff
@@ -191,7 +191,7 @@ public async Task UploadWithMissingCoordinatesAsync()
         ValidationProblemDetails problemDetails = (ValidationProblemDetails)result.Value!;
```

Comment:
Q: Haben wir ein Testfall mit einem File, bei dem erst in einer späteren Zeile ein leeres Feld erscheint? Analog zum beschriebenen Test von Renato in https://github.com/geoadmin/suite-bdms/issues/551#issuecomment-1510768188. Falls nicht, wäre solch ein Testfall sinnvoll.

Reply:
Das stimmt. So ein Test habe wir. Der Test _UploadMulitpleRowsMissingRequiredFieldsShouldReturnError()_ simuliert 2 Fehler. Auf Zeile 2 und 3. Zeile 1 ist valid.